modules/odc-deploying-java-applications.adoc

```diff
@@ -2,40 +2,42 @@
 
 * Navigate to the *+Add* view of the *Developer* perspective, and click *Upload JAR file* in the *From Local Machine* tile. Browse and select your JAR file, or drag and drop a JAR file to deploy your application.
 
 * Navigate to the *Topology* view and use the *Upload JAR file* option, or drag and drop a JAR file to deploy your application.
 
 * Use the in-context menu in the *Topology* view, and then use the *Upload JAR file* option to upload your JAR file to deploy your application.
 
-Use the following instructions to upload a JAR file in the *Topology* view to deploy a Java application:
+.Prerequisites
+
+* The Cluster Samples Operator must be installed by a cluster administrator.
+* You have access to the {product-title} web console and are in the *Developer* perspective.
 
 .Procedure
 
-. In the *Topology* view, right-click anywhere in the *Topology* view to see the *Add to Project* menu.
+. In the *Topology* view, right-click anywhere to view the *Add to Project* menu.
 
-. Hover over the *Add to Project* menu to see the menu options, and then select the *Upload JAR file* option to see the *Upload JAR file* form . Alternatively, you can drag and drop the JAR file in the *Topology* view.
+. Hover over the *Add to Project* menu to see the menu options, and then select the *Upload JAR file* option to see the *Upload JAR file* form. Alternatively, you can drag and drop the JAR file in the *Topology* view.
 
 . In the *JAR file* field, browse for the required JAR file on your local machine and upload it. Alternatively, you can drag and drop the JAR file on the field. A toast alert is displayed at the top right if an incompatible file type is dragged and dropped on the *Topology* view. A field error is displayed if an incompatible file type is dropped on the field in the upload form.
 
-. You can further specify optional Java commands to customize your deployed application. The Runtime Icon and Builder Image is selected by default. If a Builder Image is not auto-detected, select a Builder Image. If required, you can change the version using the *Builder Image Version* drop-down list.
+. The runtime icon and builder image are selected by default. If a builder image is not auto-detected, select a builder image. If required, you can change the version using the *Builder Image Version* drop-down list.
 
-. In the optional *Application Name* field, enter a unique name for your application for the resource labelling.
+. Optional: In the *Application Name* field, enter a unique name for your application to use for resource labelling.
 
 . In the *Name* field, enter a unique component name for the associated resources.
 
-. Optional: Using the *Resource type* drop-down list, select a different resource type from the list of default resource types. 
+. Optional: Use the *Advanced options* -> *Resource type* drop-down list to change the resource type. 
```

Comment:
```suggestion
. Optional: Use the *Resource type* drop-down list to change the resource type. 
```

The Resource type isn't part of the Advanced options anymore. It looks like this:

![image](https://github.com/openshift/openshift-docs/assets/139310/5042cbf5-9997-4ad0-8236-06bb1a3a772b)

Reply:
Thank you! I didn't originally write the docs for this so it's good for this to be corrected.
Do you know which version of OCP this changed in?